apps/package.json

```diff
@@ -83,7 +83,7 @@
     "ejs": "0.8.4",
     "ejs-compiled-loader": "2.1.1",
     "ejs-loader": "0.2.1",
-    "enzyme": "2.4.1",
+    "enzyme": "^2.9.1",
```

Comment:
Won't `yarn.lock` take care of pinning?  Trying to understand when I should use `^1.0.0` vs. a specific version.